extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -96,20 +96,16 @@ long getPollRetryMs()
     try {
       Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
 
+      final Map<String, Object> consumerConfigs = KafkaConsumerConfigs.getConsumerProperties();
       final Properties props = new Properties();
+      props.putAll(consumerConfigs);
```

Comment:
This should be called after `KafkaRecordSupplier.addConsumerPropertiesFromConfig`

Reply:
ah ok, yeah i was wondering if the order matters.